bin/README.md

```diff
@@ -0,0 +1,26 @@
+## Bin
+
+This directory contains a number of scripts for running pathway parameter advising, updating reference pathways, and installing PGD.
+
+Most of these scripts must be run inside the `bin` directory. 
+
+## Files
```

Comment:
Can we add the expected input arguments for the scripts?

Reply:
Added arguments to bin/README.md